### PR TITLE
Ta-464 adding accessibility to remaining pages

### PR DESCRIPTION
**Pull request by marianadpereira**
https://tools.hmcts.net/jira/browse/TA-464

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/pages/events/enterAllocationProposalEvent.page.js [hunk 1]
```diff
@@ -9,11 +9,11 @@ module.exports = {
 
   async selectAllocationProposal(proposal) {
     I.waitForElement(this.fields.allocationProposalRadioGroup);
-    await I.runAccessibilityTest();
     I.click(proposal);
+    await I.runAccessibilityTest();
   },
 
-  enterProposalReason(reason) {
+  async enterProposalReason(reason) {
```
Comment (tomaszpowroznik) [inline e2e/pages/events/enterAllocationProposalEvent.page.js:16]: no need for async here

File: e2e/pages/events/enterApplicantEvent.page.js [hunk 2]
```diff
@@ -39,10 +39,9 @@ module.exports = {
     I.fillField(this.fields.applicant(applicantIndex).mobileNumber, applicant.mobileNumber);
     I.fillField(this.fields.applicant(applicantIndex).jobTitle, applicant.jobTitle);
     I.fillField(this.fields.applicant(applicantIndex).email, applicant.email);
-    await I.runAccessibilityTest();
   },
 
-  enterSolicitorDetails(solicitor) {
+  async enterSolicitorDetails(solicitor) {
```
Comment (tomaszpowroznik) [inline e2e/pages/events/enterApplicantEvent.page.js:44]: same

File: e2e/pages/events/enterGroundsForApplicationEvent.page.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ module.exports = {
     I.fillField(this.fields.thresholdDetails, 'mock threshold details');
   },
 
-  enterGroundsForEmergencyProtectionOrder() {
+  async enterGroundsForEmergencyProtectionOrder() {
```
Comment (tomaszpowroznik) [inline e2e/pages/events/enterGroundsForApplicationEvent.page.js:24]: same

File: e2e/pages/caseList.page.js [hunk 4]
```diff
@@ -46,10 +46,10 @@ module.exports = {
   async searchForCasesWithName(caseName, state='Any') {
     this.setInitialSearchFields(state);
     // wait for our filters to load
-    I.waitForVisible(this.fields.caseName, 30);
+    await I.waitForVisible(this.fields.caseName, 30);
+    await I.fillField(this.fields.caseName, caseName);
+    await I.click(this.fields.search);
```
[the remark this answers is not in the document]
Comment (marianadpereira) [reply e2e/pages/caseList.page.js:51]: This needs to stay like this, else this page doesn't run the accessibility test.

File: e2e/pages/events/addApplicationDocumentsEvent.page.js [hunk 5]
```diff
@@ -11,11 +11,12 @@ module.exports = {
   },
 
   async addApplicationDocument(option, file, name, description) {
+
     await I.addAnotherElementToCollection('Documents');
-    await I.runAccessibilityTest();
     const index = await I.getActiveElementIndex();
 
-    this.selectDocumentType(option, index);
+    await this.selectDocumentType(option, index);
```
[the remark this answers is not in the document]
Comment (marianadpereira) [reply e2e/pages/events/addApplicationDocumentsEvent.page.js:18]: This needs to stay like this, else this page doesn't run the accessibility test.

**Review by tomaszpowroznik — APPROVED** [review]

**Pull request merged**